docs/reference/security/authorization/privileges.asciidoc

```diff
@@ -73,14 +86,39 @@ owned by other users.
 
 --
 
+`manage_autoscaling`::
+All operations related to managing autoscaling policies.
+NOTE: This privilege is not available in {serverless-full}.
+
+// manage_behavioral_analytics:: TBD
```

Comment:
This one is available in both serverless and stateful. 

This privilege was recently added by the Enterprise Search team. I think the description for it could be something like "All operations related to managing behavioral analytics". To keep things focused, I can open a separate PR and get a review from Ent Search.

Reply:
That sounds like a good idea, thanks

Reply:
👍 I think there's a rendering glitch because of the comment above though (see attached). I've created a Jira task (ES-8902) to track adding docs for the missing privileges so it's safe to remove TBD.  

<img width="877" alt="Screenshot 2024-07-03 at 14 20 33" src="https://github.com/elastic/elasticsearch/assets/13320119/fa44a270-a80e-4c4f-8403-d74691da71c6">

Reply:
Thanks for catching that glitch! I removed the comments in 439e8ef4b28ba00f3a2d728107536ec637509285